### ISSUE TITLE
How should species estimates be managed?

### ISSUE THREAD

**Issue by mdoering** [Sp2000/colplus-frontend#128]
Species estimates are an estimation count together with a source reference. They usually are tight to higher taxa, families and above. But may also be about a genus.

Estimates in theory could come from the sources, but so far it is only the CoL editor that manages these estimates. I expect members from the CoL taxonomic group to also manage species estimates for the CoL as they have been doing this in parallel already.

Although the taxon class currently holds the estimates properties they cannot be modified directly on the taxon records as these would be removed when a sector is updated. They should rather be linked to CoL taxon ids in the same way we deal with editorial decisions. 

@thomasstjerne is a dedicated tool needed to manage the estimates or can this be incorporated into the assembly section as a new tab/pill ?

**Comment by thomasstjerne** [comment]
I was considering to have species estimates in the assembly tree (when in modify mode).

**Comment by mdoering** [comment]
duplicate of #3
